src/include/uapi/ipc.h

```diff
@@ -240,6 +240,7 @@ struct sof_ipc_compound_hdr {
 #define SOF_DAI_INTEL_SSP_QUIRK_PSPSRWFDFD	(1 << 5)
  /* here is the possibility to define others aux macros */
 
+#define SOF_DAI_INTEL_SSP_FRAME_PULSE_WIDTH_MASK	0x3F
```

Comment:
I'd use a max value instead of a mask to better detect topology issues and range.